db/migrate/20231122105957_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.1]
+  def self.up
```

Comment:
- [ ] When I try to run your migration files, I get this error:

![image](https://github.com/ahmidou09/blog_app/assets/77106962/bf605772-6d61-452c-bf4d-f6b73cda3934)

The migration files that add indexes to tables should come after the migration files that create those tables. For example, the migration file for creating the `Likes` table should come before the migration file that adds indexes to the `Likes` table. 

I would advise you to rename the migration files in such a way that the migration files for creating tables appear before those that add indexes in the _**db/migrate**_ directory. 👍 😎 

It is important for your migration files to run correctly. It will make contributors have access to the project without hassles.

#### Great work implementing all changes suggested by previous reviewers. If you implement this change, without altering any other file, then you are good to go.